bluetooth: mesh: Fix Light PI regulator-related issues. (#9865)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

Bluetooth mesh
--------------

> * Fixed:

Reviewer: Also wondering if these two changes should rather go in under libraries.

Author: As this is a bug-fix for existing features, I think it is correct where it is. It seems, only new feature additions are reported in the library section.

Reviewer: Oki, we leave it there

> * Fixed:
> 
>   * An issue in :ref:`bt_mesh_light_ctrl_srv_readme` model where multiple scene recall messages for the same scene will not repeately trigger same scene recall. This helps in preventing ongoing transition interruption.

Reviewer: ```suggestion
* Fixed:

  * An issue in the :ref:`bt_mesh_light_ctrl_srv_readme` model where multiple scene recall messages for the same scene do not repeatedly trigger the same scene recall.
    This prevents the interruption of an ongoing transition.
```

* Updated:

>   * :ref:`bt_mesh_light_ctrl_srv_readme` model to make sure when controller is enabled and disabled, the regulator starts running when fresh value of ambient lux level is reported.

Reviewer: Not sure I understand the sentence...

Author: Hmm, 'disabled' is probably not needed. How about this?
`... make sure when the controller is enabled, the regulator starts running when a fresh value of ambient lux level is reported.`

Reviewer: Also, is it clear what regulator we are talking about, or should we state illuminance regulator?

Reviewer: And should it then be LuxLevel or still lux level?
Ref. the other comment in the light_ctrl_srv.c file.

  * Updated:

    * Enabled :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` configuration option.

>     * The specification-defined lightness PI regulator (:kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_REG_SPEC`) now selects :kconfig:option:`CONFIG_FPU` and thus doesn't require it to be enabled explicitly in the project file.

Reviewer: ```suggestion
    * The specification-defined illuminance regulator (:kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_REG_SPEC`) now selects the :kconfig:option:`CONFIG_FPU` option by default, thus it is not required to enable it explicitly in the project file.
```
